helix-core/src/main/java/org/apache/helix/task/FixedTargetTaskAssignmentCalculator.java

```diff
@@ -78,11 +78,11 @@ public FixedTargetTaskAssignmentCalculator(AssignableInstanceManager assignableI
 
   @Override
   public Map<String, SortedSet<Integer>> getTaskAssignment(CurrentStateOutput currStateOutput,
```

Comment:
So it may not be safe to change public API directly. My suggestion is to deprecate the method and create a new one if necessary. 

This deprecated function can call the new function.

Reply:
Done.